### DISCUSSION TITLE
1080p Premium Quality Option for free?

### DISCUSSION THREAD

**Discussion by gototheskinny** [arichornlover/uYouEnhanced#186] [Ideas]
Smarttube has this option and I love it. I'm not telling "is it possible" bc it's for other places, Would you consider this as an **_option_** in the app because I know youtube would come and destroy this repo with a quickness  if it was enabled by default. Smarttube don't even **mention** this option or **ability** as I would call it in their Repo.

**Comment by sfksuperman** [comment]
Share the smarttube ipa file bro. Thanks.

**Reply by gototheskinny** [reply]
It's not an ipa. It's an android tv app. It's an apk. Search smarttube on Google

**Reply by sfksuperman** [reply]
Lol, my bad!
